doc/sai-ptf/fdb_test_plan.md

```diff
@@ -0,0 +1,264 @@
+### Case6: test_flush_all_dynamic
+### Case7: test_flush_all
+
+### Testing Objective <!-- omit in toc -->
+Verify flushing of static/dynamic entries on VLAN/Port/All.
+### Test steps: <!-- omit in toc --> 
+- test_flush_vlan_static
+- test_flush_port_static
+- test_flush_all_static
+
+1. Flush with conditions for each case: ``Static`` flush on ``VLAN10``; ``Static`` flush on ``Port1``; flush for all ``Static`` 
+2. Send packets for each case in sequence: ``port1`` DMAC=``Port2 MAC``; ``Port2`` DMAC=``Port1 MAC``; ``port1`` DMAC=``Port2 MAC``
+3. Verify flooding happened, packets received in related VLAN, except the ingress port.
+4. Send packets for each case in sequence:  ``Port9`` DMAC=``Port10 MAC``; ``Port10`` DMAC=``Port9 MAC``; `  ``Port9`` DMAC=``Port10 MAC``
+5. Verify flush happens in a certain domain, unicast to the corresponding port.
+
+- test_flush_vlan_dynamic
+- test_flush_port_dynamic
+- test_flush_all_dynamic
+  
+1. Flush with conditions for each case in sequence: ``Dynamic`` flush on ``VLAN20``;  ``Dynamic`` flush on ``Port9``; flush for all ``Dynamic`` 
+2. Send packets for each case in sequence: ``Port9`` DMAC=``Port10 MAC``;  ``Port10`` DMAC=``Port9 MAC``; ``Port9`` DMAC=``Port10 MAC``
+3. Verify flooding happened, packets received in related VLAN, except the ingress port.
+4. Send packets for each case in sequence:  ``port1`` DMAC=``Port2 MAC``;``Port2`` DMAC=``Port1 MAC``; ``port1`` DMAC=``Port2 MAC``;
+5.  Verify flush happens in a certain domain, unicast to the corresponding port.
+
+- test_flush_all
+
+1. Flush with conditions: flush for ``All``
+2. Send packets : ``port1`` DMAC=``Port2 MAC``; ``Port9`` DMAC=``Port10 MAC``; 
+3. Verify flooding happened, packets received in related VLAN, except the ingress port.
+
+
+## Test Group4: MAC move
+### Case1: test_disable_move_drop
+### Case2: test_dynamic_mac_move
+### Case3: test_static_mac_move
+### Testing Objective <!-- omit in toc --> 
+- test_disable_move_drop: Verify if disable MAC move, drop packet with known SMAC if the SMAC was already learnt on other port.
+- test_dynamic_mac_move: Verify when enabling MAC move, previous learnt mac(SMAC) on a port can be learnt on other port
+- test_static_mac_move: Verify when enabling MAC move, previous installed mac(static SMAC) on a port can be set to other port
+
+### Test steps: <!-- omit in toc --> 
+- test_disable_move_drop
+
+1. Disable mac move for ``Port1 MAC`` on Port1
+2. Create a packet with SMAC=``Port1 MAC`` DMAC=``Port2 MAC``
+3. Send packet on port1
+4. Verify packet received on port2
+5. Send packet in step2 on port3
+6. Verify the packet gets dropped
+
+- test_dynamic_mac_move
+  
+1. Flush All MAC
+2. Install static FDB entry for port2 with ``Port2 MAC``
+3. Send Packet on Port1 with SMAC=``Port1 MAC`` DMAC=``Port2 MAC``
+4. Create packet with SMAC=``Port1 MAC`` DMAC=``Port2 MAC``
+5. Send packet on port1
+6. Install static mac move for ``Port1 MAC`` on Port1 and enable mac move
```

Comment:
why install static mac move, since this one is for dynamic mac move?

Reply:
yes, mac move attribute test. add more description for this test